src/views/preview/project-view.jsx

```diff
@@ -665,6 +673,7 @@ const mapStateToProps = state => {
             state.permissions.admin === true),
         isLoggedIn: isLoggedIn,
         isAdmin: isAdmin,
+        isNewScratcher: isLoggedIn && state.permissions.new_scratcher,
```

Comment:
Good point. Yeah, as helpful as these individual flags (and the `can`- flags) are, they can be a little misleading -- like how it's hard to know exactly what `canCreateNew` means, since in a sense, any user can create new at any time.

It could be better to give these more elaborate names, such as `isLoggedInNewScratcher`, but even for me (who prefers verbosity more than most!) that seems like overkill.

So I appreciate your noting this.